app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -186,5 +154,69 @@
         });
       }
     }
+
+    /*
+     * Determins if the item has damange and it's non kinetic.
+     *
+     * @param {Object} item - A item from the destiny server
```

Comment:
Do you guys find jsdocs helpful for this project? I'm used to always including it, but not sure how to properly describe the item @param.